tests/upgrades/test_remoteexecution.py

```diff
@@ -45,18 +43,18 @@ class Scenario_remoteexecution_external_capsule(APITestCase):
 
     @classmethod
     def setUpClass(cls):
-        cls.libvirt_vm = os.environ.get('LIBVIRT_HOSTNAME')
+        cls.libvirt_vm = settings.compute_resources.libvirt_hostname
         cls.default_org_id = entities.Organization().search(
             query={'search': 'name="{}"'.format(DEFAULT_ORG)})[0].id
         cls.org = entities.Organization(id=cls.default_org_id).read()
-        cls.bridge = os.environ.get('BRIDGE')
-        cls.subnet = os.environ.get('SUBNET')
-        cls.gateway = os.environ.get('GATEWAY')
-        cls.netmask = os.environ.get('NETMASK')
-        cls.vm_domain_name = os.environ.get('VM_DOMAIN')
+        cls.bridge = settings.vlan_networking.bridge
+        cls.subnet = settings.vlan_networking.subnet
+        cls.gateway = settings.vlan_networking.gateway
+        cls.netmask = settings.vlan_networking.netmask
+        cls.vm_domain_name = settings.upgrade.vm_domain
         cls.vm_domain = entities.Domain().search(query={'search': 'name="{}"'
                                                  .format(cls.vm_domain_name)})
-        cls.proxy_name = os.environ.get('RHEV_CAP_HOST')
+        cls.proxy_name = settings.upgrade.rhev_cap_host or settings.upgrade.capsule_hostname
 
```

Comment:
`capsule_hostname` exist in robottelo.properties file but need to be sed while running the pipeline, this currently not exists in  https://github.com/SatelliteQE/robottelo-ci/blob/c89f0f2026e94e02672086b90dbebb1b983157a4/scripts/satellite6-upgrade-run-scenarios.sh#L19

Reply:
This is a completely optional field and you can only use that field while running the tests locally on your local capsule.

In short, its either RHEV_CAP_HOST or CAPSULE_HOSTNAME

Reply:
what if RHEV_CAP_HOST does not exist? you want to use CAPSULE_HOSTNAME if that is never a case then we can make it like this 

`cls.proxy_name = settings.upgrade.rhev_cap_host`

Reply:
RHEV_CAP_HOST is there always when we run tests on Jenkins as we run upgrades on RHEV based images.

Reply:
this means we don't require `CAPSULE_HOSTNAME` as an additional/optional argument, this where I'm pointing. If that is not true do we set  `CAPSULE_HOSTNAME` in the pipeline anywhere ?.

Reply:
you can only use that field while running the tests locally on your local capsule. - sorry I missed this message. In this case, looks ok then.